pkgs/test_api/lib/backend.dart

```diff
@@ -2,6 +2,10 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+@Deprecated('package:test_api is not intended for general use. '
+    'Please use package:test.')
+library test_api;
```

Comment:
`test_api.backend`.

I don't know if there is a requirement for unique library names - but we should aim for it.

Reply:
Yeah I wasn't sure if having the same library name would be problematic but it didn't appear to cause issues. Updated it nonetheless.